CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,11 @@ Fix: Error message when sending measures with unknown/undefined attribute
 Add Null check within executeWithSecurity() to avoid crash (#829)
 Add NGSIv2 metadata support to attributeAlias plugin.
 Add mongodb authentication: IOTA_MONGO_USER, IOTA_MONGO_PASSWORD and IOTA_MONGO_AUTH_SOURCE (#844)
+Add basic NGSI-LD support as experimental feature (#842)
```

Comment:
I have done a little refactor to the NGIS-LD entries in the CNR file.

The line "Lazy attributes" is mine but I guess is correct, as in https://github.com/telefonicaid/iotagent-node-lib/issues/842#issuecomment-583404601I see Lazy Attribute checked.

@jason-fox is my understanding correct?

Reply:
Yes - Lazy Attributes are mapped to the forwarded GET

Reply:
Thus, NTC (I guess)